sdk/identity/identity/test/manual-integration/AzureVM/index.ts

```diff
@@ -0,0 +1,18 @@
+import { SecretClient } from "@azure/keyvault-secrets";
+import { DefaultAzureCredential } from "@azure/identity";
+
+async function main(): Promise<void> {
+  // This will use the system managed identity
+  const credential = new DefaultAzureCredential();
+
+  const url = "https://" + process.env.KEY_VAULT_NAME + ".vault.azure.net";
+  const client = new SecretClient(url, credential);
+
+  await client.setSecret("secret-name-system", "secret-value-system");
```

Comment:
I just want to acknowledge that I understand that you're fine by not testing the result. Checking that things *just* work is indeed appropriate for this kind of testing.